routes/api/contacts.js

```diff
@@ -1,25 +1,77 @@
-const express = require('express')
+const express = require("express");
+const router = express.Router();
+const contactsController = require("../../controllers/contacts");
+const authMiddleware = require("../../middleware/authMiddleware");
 
-const router = express.Router()
+router.get("/", authMiddleware, async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const contacts = await contactsController.getAll(userId);
+    res.json(contacts);
+  } catch (error) {
+    res.status(500).json({ message: "Error fetching contacts" });
+  }
+});
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/:id", authMiddleware, async (req, res) => {
+  const contactId = req.params.id;
+  const userId = req.user.id;
+  try {
```

Comment:
same here. You have contacts controller file. But still got all functions inside routes. What's the point of having controller then?

Reply:
your routes should look like:
`router.get("/:id", authMiddleware, getUserById)`

and that's all